.travis.yml

```diff
@@ -40,14 +40,19 @@ install:
         cd ..
         mkdir -p catkin_ws/src
         mv opw_kinematics catkin_ws/src/
-        cd catkin_ws
+        cd catkin_ws        
         choco uninstall mingw
+        wget https://raw.githubusercontent.com/johnwason/source_bat/master/source_bat.bash
+        source source_bat.bash 'C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat'
         choco source add -n=ros-win -s="https://roswin.azurewebsites.net/api/v2" --priority=1
         choco install ros-melodic-ros_base -y --execution-timeout=0
-        wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
-        cmd "/C call travis_ci_windows.bat install "
+        #wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
+        #cmd "/C call travis_ci_windows.bat install "        
+        source source_bat.bash 'c:\opt\ros\melodic\x64\setup.bat'
+        rosdep install --from-paths src --ignore-src -r -y
+        /c/opt/vcpkg/vcpkg.exe install jsoncpp:x64-windows
     fi
 
 script:
   - if [ "$TRAVIS_OS_NAME" != "windows" ]; then .ci_config/travis.sh; fi
-  - if [ "$TRAVIS_OS_NAME" = "windows" ]; then cmd "/C call travis_ci_windows.bat build "; fi
+  - if [ "$TRAVIS_OS_NAME" = "windows" ]; then catkin_make_isolated -DENABLE_TESTS=ON; fi
```

Comment:
I looked at industrial ci and probably want to also pass the following two cmake arguments `-DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS_RELEASE="-O3"`

Reply:
I set the build type to `Release`. Setting `-O3` is unnecessary as the default flags for `Release` already do so.